Sources/App/Slack/Message.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Message.swift
+//  App
+//
+//  Created by Allen Humphreys on 3/5/19.
+//
+
+import SlackKit
+
+// https://api.slack.com/messaging/managing
+// https://api.slack.com/messaging/sending#threading
+// https://api.slack.com/methods/chat.postMessage
+
+/// Represents a message from Slack. We could very easily create different types for
+/// incoming and outgoing messages to avoid having optional properties
+class Message {
+    let text: String
+    let channelID: ChannelID
+    let parent: String?
+
+    // I don't know that it makes sense to receive these?
+    var attachments = [Attachment]()
+
+    // Sender is only really useful for messages we've received
+    var sender: String? = nil
```

Comment:
Could we just keep these `:String?` instead of initializing them to nil ? Seems unnecessary

Reply:
At some point I had these as `let` and I added this to shut the compiler up